### PR TITLE
Implement :length policy

### PR DESCRIPTION
**Pull request by belinskidima**

### CODE REVIEW

File: lib/paradocs/policies.rb [hunk 1]
```diff
@@ -139,4 +139,27 @@ def ok?(options, actual)
       [actual].flatten.all?{|v| options.include?(v)}
     end
   end
+
+  Paradocs.policy :length do
+    COMPARISONS = {
+      max: [:>, "maximum"],
+      min: [:<, "minimum"]
```
Comment (mtkachenk0) [inline lib/paradocs/policies.rb]: add equal
```rb
eq: 5
```

File: README.md [hunk 2]
```diff
@@ -224,6 +224,22 @@ field(:status).options(["draft", "published"])
 field(:status).policy(:options, ["draft", "published"])
 ```
 
+### :length
+
+Specify value's length constraints
+ - `min:` - The attribute cannot have less than the specified length.
+ - `max`  - The attribute cannot have more than the specified length.
+ - `eq`   - The attribute should be exactly than the specified length.
```
Comment (mtkachenk0) [inline README.md]: did you mean: exactly equal to the specified length?

File: README.md [hunk 3]
```diff
@@ -224,6 +224,22 @@ field(:status).options(["draft", "published"])
...
+
+```ruby
+field(:name).length(min: 5, max: 25)
+field(:name).length(eq: 10)
+
+# Same as
```
Comment (mtkachenk0) [inline README.md]: this section is redundant

File: lib/paradocs/policies.rb [hunk 4]
```diff
@@ -139,4 +139,28 @@ def ok?(options, actual)
...
+        obj << "#{COMPARISONS[comparison].last} #{limit} characters"
+      end.join(", ")
+    end
+
+    validate do |options, actual, key, payload|
+      !payload.key?(key) || ok?(options, actual)
+    end
+
+    def ok?(options, actual)
+      options.reject do |comparison, limit|
```
Comment (mtkachenk0) [inline lib/paradocs/policies.rb]: instead of reject + empty? I'd recommend to use `all?`

File: README.md (hunk 2 above)
Comment (belinskidima) [reply README.md]: yep )

**Pull request merged**